bootstrap/eks/internal/userdata/node.go

```diff
@@ -0,0 +1,17 @@
+package userdata
+
+const (
+	nodeUserData = `#!/bin/bash
+/etc/eks/bootstrap.sh {{.ClusterName}}
+`
+)
+
+// NodeInput defines the context to generate a node user data.
+type NodeInput struct {
+	ClusterName string
+}
+
+// NewNode returns the user data string to be used on a node instance.
+func NewNode(input *NodeInput) ([]byte, error) {
+	return generate("Node", nodeUserData, input)
```

Comment:
Only one line here. Maybe we can move the implementation here and get rid of the file below?

Reply:
Yup, I agree.

Reply:
Done.